src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationAnnotationIncorrect.java

```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+class InpuIndentationAnnotationIncorrect { //indent:0 exp:0
```

Comment:
https://djydewang.github.io/diffReport_Issue5154/checkstyle/xref/home/bbg/project/contribution/checkstyle-tester/repositories/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationIncorrect.java.html#L7
Test case we mentioned before had annotation on top class. Can you replicate it in this input file. You may leave the annotation on the nested class as an additional case.

Reply:
Done.